packages-common/pluggable-widgets-generator/generators/app/lib/text.js

```diff
@@ -0,0 +1,50 @@
+const chalk = require("chalk");
+const { join } = require("path");
+
+const pkg = require(join(__dirname, "../../../package.json"));
+
+module.exports = {
+    BANNER: `
+
+    ${chalk.bold.bgBlueBright("  __  ____   __ ")}           _     _            _    
```

Comment:
is this string actually ok?

Reply:
Yes, checked it. Text and logo look nice.